src/utils/response/responseHandlers.ts

```diff
@@ -14,7 +14,7 @@ export async function toApiResponse<T>(promise: RequestPromise<T>): Promise<ApiR
   } catch (e) {
     const error = e as AxiosError;
 
-    if (error.response?.data != null && 'Success' in error.response.data) {
+    if (error.response?.data != null && typeof error.response.data !== 'string' && 'Success' in error.response.data) {
```

Comment:
Should we maybe check if it's an object instead? Just to be sure that `in` doesn't throw
```suggestion
    if (error.response?.data != null && typeof error.response.data === 'object' && 'Success' in error.response.data) {
```

Reply:
That was my initial idea, but it broke 9 or so tests.
Didn't look too much into them, but a guess from the top of my head is that it can also be an array and, maybe, a couple of other things that we've overlooked.
With that in mind, i decided to switch to explicitly check for a `string`, because that's a case that we know for sure (1) can happen and (2) will break things.